CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ All notable changes to the Wazuh app for Splunk project will be documented in th
 - Fixed agent's name overflow in the overview [#1137](https://github.com/wazuh/wazuh-splunk/pull/1137)
 - Fixed unnecessary table requests when resizing browser window [#1141](https://github.com/wazuh/wazuh-splunk/pull/1141)
 - Fixed on save rules or decoders files [#1138](https://github.com/wazuh/wazuh-splunk/pull/1138)
+- fixed overlapping security options [#1217](https://github.com/wazuh/wazuh-splunk/pull/1217)
```

Comment:
Make it consistent and understandable. 

1. Typo on the first letter: fixed --> Fixed
2. Fix what? --> "Fixed the underlapping navigation bar for Security options"